ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -0,0 +1,164 @@
+import os
+import logging
+import shutil
+import tempfile
+import re
+from pathlib import Path
+
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.ocs.utils import collect_ocs_logs
+from ocs_ci.ocs.must_gather.const_must_gather import (
+    GATHER_COMMANDS_CEPH,
+    GATHER_COMMANDS_JSON,
+    GATHER_COMMANDS_OC_OUTPUT
+)
+
+
+logger = logging.getLogger(__name__)
+TYPE_LOG = {
+    'CEPH': GATHER_COMMANDS_CEPH,
+    'JSON': GATHER_COMMANDS_JSON,
+    'OC_OUTPUT': GATHER_COMMANDS_OC_OUTPUT
+}
+
+
+class MustGather(object):
+    """
+    MustGather Class
+
+    """
+    def __init__(self):
+        self.type_log = None
+        self.root = None
+        self.files_path = dict()
+        self.empty_files = list()
+        self.files_not_exist = list()
+        self.files_content_issue = list()
+
+    @property
+    def log_type(self):
+        return self.num_of_builds
```

Comment:
I don't see `self.num_of_builds` instantiated anywhere, why is this being returned for the `log_type` property?